corehq/apps/domain/templates/domain/admin/manage_alerts.html

```diff
@@ -76,10 +76,10 @@ <h3>
               </form>
             </td>
             <td>
-              <a class="btn btn-default" data-bind="attr: {href: editUrl}">
+              <a class="btn btn-default" data-bind="attr: {href: editUrl}" style="float: left; margin-right: 5px;">
```

Comment:
I believe the recommendation is to not add styling in html, though I am not sure if it's reasonable to add a new file just for this view here. Happy to reconsider this if needed.

Reply:
ya I did try those and they didn't apply.

Reply:
Yeah, our guidance is not to use inline styles. Why not put the edit and delete buttons in separate columns? That's simpler than adding custom styling here, and it's more semantic since the markup (and therefore the screen reader experience) will match the visual experience rather than flipping the button order. I'd also remove the "Activate or De-activate" and "Actions" headers, which are redundant with the button text.

FWIW, Zandre's class suggestions didn't work because this is a bootstrap 3 page. B3 has `pull-left` but doesn't have margin utilities.

Reply:
Cool, thanks @orangejenny 
I will check with others if the change to the look works okay and then make changes accordingly.

Though I like how the buttons are grouped as of now and might need to retain styling on these elements, should I add a new CSS file for this page? Is this process documented somewhere?

Before:
![Screenshot from 2024-01-31 16-47-57](https://github.com/dimagi/commcare-hq/assets/3864163/a6c7096d-020b-458d-8333-e51ff7f1952e)

After:
![Screenshot from 2024-01-31 16-48-08](https://github.com/dimagi/commcare-hq/assets/3864163/57a107bc-0146-4f2e-ba4c-702485911dd1)

Reply:
This page shouldn't have custom styling for just this reason. There are other pages with similar tables and columns of buttons, and we should be consistent. If you'd like to look for a more standard way to get different spacing, check out button groups/toolbars [here](https://getbootstrap.com/docs/3.3/components/#btn-groups).

Reply:
Follow up:

https://github.com/dimagi/commcare-hq/pull/34138
> There are other pages with similar tables and columns of buttons, and we should be consistent

Keeping in line with how [invitations listing works for web users](https://staging.commcarehq.org/a/ccqa-mk/settings/users/web/), kept the edit and delete button in the same column. Additionally, putting them in separate columns, got them really far apart (as was shown in the 2nd screenshot above).

Tried using the button groups, but then I loose the button rounded edges. Additionally, it does not seem to work here, which I guess is because I have one anchor and one form button. So retaining the current structure.